src/BinSkim.Driver/AnalyzeCommand.cs

```diff
@@ -52,7 +52,11 @@ public override PropertyBag CreateConfigurationFromOptions(AnalyzeOptions analyz
         protected override void AnalyzeTarget(IEnumerable<ISkimmer<BinaryAnalyzerContext>> skimmers, BinaryAnalyzerContext context, HashSet<string> disabledSkimmers)
         {
             base.AnalyzeTarget(skimmers, context, disabledSkimmers);
-            AnalyzeManagedAssembly(context.TargetUri.LocalPath, _plugInFilePaths, context);
+
+            if (context.PE.IsManaged && !context.PE.IsManagedResourceOnly)
```

Comment:
1. Don't you want to emit a "not checked because not managed" result in the SARIF?
2. May be a good idea to use Roslyn's bits to check for "is managed" over the "try to parse the method table" garbage currently in Dia2Lib.